### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the listitems in the table in "Settings  dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1497]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin tree item.
4. Tab to list items under the table and observe the Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA are not announcing the name of the list items present in the table.

**Jaws Behaviour:**
Jaws is not announcing list items present in the table.

**NVDA Behaviour:**
selected row 5 column 1
selected row 6 column 1

**Expected Result:**
Jaws and Narrator should announce the name of the list items present in the table

Jaws and NVDA should announce as
copyright selected row 5 column 1
coverage selected row 6 column 1

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![settings-list items](https://user-images.githubusercontent.com/22956130/36136607-67f02a9e-10b7-11e8-97ef-f4cdf78a8eed.jpg)
![ec705f93](https://user-images.githubusercontent.com/22956130/36136616-6dc07ae6-10b7-11e8-8711-2a4badd6c21a.jpg)
![settings-listitems-explorer](https://user-images.githubusercontent.com/22956130/36136621-70a5a010-10b7-11e8-9dde-6de1492a9c8e.jpg)

**Comment by Eskibear** [comment]
Content is provided by IntelliJ settings dialog, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749945
